### PR TITLE
homepage

### PR DESCRIPTION
**Pull request by ranitasahaa1230**
Added homepage
Added category wise quiz
Link:
https://deploy-preview-1--quizomaniac-hub.netlify.app/

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,139 @@
...
-.App-link {
-  color: #61dafb;
+::-webkit-scrollbar-thumb {
+  background-color: var(--secondary-color);
+}
+
+/* .primary-text {
+  color: var(--text-color);
+  font-size: 1.2rem;
+} */
```
Comment (rohanmathur91) [inline src/App.css:56]: Remove this if not needed

File: src/components/Categories/Categories.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+              </div>
+              <div className="card__desc">
+                <div className="desc__span">10 Questions</div>
+                <Link to="/category"><button className="text">TAKE QUIZ</button></Link>
+              </div>
+                </div>
+              ))}
+        </div>
+        </main>
+  )
```
Comment (rohanmathur91) [inline src/components/Categories/Categories.js:25]: run prettier in this file

File: src/components/Categories/data.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      description: "",
+      img:"",
+      alt: "",
+    },
+    // {
+    //   id: 5,
+    //   item: "gloves",
+    //   quantity: 1
+    // }
+  ];
```
Comment (rohanmathur91) [inline src/components/Categories/data.js:35]: remove this

File: src/App.css [hunk 4]
```diff
@@ -1,38 +1,139 @@
...
   }
-  to {
-    transform: rotate(360deg);
+  section, .form-box{
+    margin-top: 18rem;
+  }
+  .main-quiz, .rules__section, .section__question, .answer__section, .profile-section,.score-board{
+    margin-top: 26rem;
   }
 }
```
Comment (rohanmathur91) [inline src/App.css:132]: check the indentation of this part

**Review by rohanmathur91 — APPROVED** [review]
Suggestion: Remove commented code if not needed
 - Good use of semantic tags
 - Code can be understood easily

File: src/components/Categories/Categories.css [hunk 5]
```diff
@@ -0,0 +1,60 @@
+main-quiz{
+    margin-top: 12rem;
+
```
Comment (Rohitprasad83) [inline src/components/Categories/Categories.css:3]: remove extra spacing

**Review by Rohitprasad83 — APPROVED** [review]
Code and UI look clean, just remove unwanted commented code.

**Review by khushijohri2001 — APPROVED** [review]
UI is so attractive and cool

**Pull request merged**